WhaleSpotting/react-app/src/styles/Map.scss

```diff
@@ -1,7 +1,7 @@
 @import "./Constants.scss";
 
 .map-component {
-    margin: $headerHeight auto $footerHeight auto;
+    margin: 0 auto $footerHeight auto;
```

Comment:
I don't think I agree with doing this, I think we should extract the margin stuff to the parent div surrounding the banner, as the banner is now too high up

Reply:
```suggestion
    margin: 0 auto;
```